pom.xml

```diff
@@ -1,24 +1,75 @@
     </properties>
-
-
-    <build>
-    </build>
-
+    <profiles>
+        <profile>
+            <id>frontend</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>com.github.eirslett</groupId>
+                        <artifactId>frontend-maven-plugin</artifactId>
+                        <version>1.12.0</version>
+                        <executions>
+                            <execution>
+                                <id>install node and npm</id>
+                                <goals>
+                                    <goal>install-node-and-npm</goal>
+                                </goals>
+                                <phase>generate-resources</phase>
+                            </execution>
+                            <execution>
+                                <id>npm install</id>
+                                <goals>
+                                    <goal>npm</goal>
+                                </goals>
+                                <phase>generate-resources</phase>
+                                <configuration>
+                                    <arguments>install</arguments>
+                                </configuration>
+                            </execution>
+                            <execution>
+                                <id>npm run build</id>
+                                <goals>
+                                    <goal>npm</goal>
+                                </goals>
+                                <configuration>
+                                    <arguments>run compile</arguments>
+                                </configuration>
+                            </execution>
+                            <execution>
+                                <id>javascript tests</id>
+                                <goals>
+                                    <goal>karma</goal>
+                                </goals>
+                                <phase>test</phase>
+                                <configuration>
+                                    <karmaConfPath>./karma.conf.single-run.js</karmaConfPath>
+                                </configuration>
+                            </execution>
+                        </executions>
+                        <configuration>
+                            <nodeVersion>v12.22.1</nodeVersion>
+                            <workingDirectory>${project.basedir}</workingDirectory>
+                        </configuration>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
+    </profiles>
     <modules>
-        <module>MapStore2/backend</module> <!-- this ensure to build the mapstore-backend module, even if not available on maven repo -->
+        <module>MapStore2/backend</module>
+        <!-- this ensure to build the mapstore-backend module, even if not available on maven repo -->
```

Comment:
```suggestion
        <!-- this ensure to build the mapstore-backend module, even if not available on maven repo -->
        <module>MapStore2/backend</module>
```
This comment refers to ther row `<module>MapStore2/backend</module>` so I put the comment above it, not below

Reply:
ok, sorry, it might have been reorder by my IDE, this wasn't intentional.